File: hu.bme.mit.incqueryd.agent/src/main/java/hu/bme/mit/incqueryd/agent/DestroyInfrastructureResource.java
package hu.bme.mit.incqueryd.agent;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.core.Response;

import com.codahale.metrics.annotation.Timed;

@Path("/destroy")
public class DestroyInfrastructureResource {

	@GET
	@Timed
	public Response execute() {
		// TODO kill processes
		return Response.ok().build();
	}

}

File: hu.bme.mit.incqueryd.agent/src/main/scala/hu/bme/mit/incqueryd/agent/DestroyInfrastructureResource.scala
package hu.bme.mit.incqueryd.agent

import com.codahale.metrics.annotation.Timed

import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.core.Response

@Path("/destroy")
class DestroyInfrastructureResource {

  @GET
  @Timed
  def execute: Response = {
    // TODO
    Response.ok.build 
  }

}
